### PR TITLE
O3-1316: Implement a feature in 3.x E2E testing tool to avoid data mutations in tests

### PR DESCRIPTION
**Pull request by pasindur99**
## Purpose
The pull request is created to implement a feature in 3.x E2E testing tool to avoid data mutations in tests. And this pull request is related to the issue ticket number [O3-1316](https://issues.openmrs.org/browse/O3-1316)

## Goals.
- This pr aim is to avoid the data mutation issues related to some tests in 3.X E2E testing tool.

## Approach
 - Create Script files for the following purposes.
         1. Take a Dump File of a database.
         2. Restore the database.
  - Create npm commands to run the following scripts.

### CODE REVIEW

File: docker/docker-compose-refqa-3x.yml [hunk 1]
```diff
@@ -53,7 +53,9 @@ services:
       MYSQL_ROOT_PASSWORD: ${MYSQL_ROOT_PASSWORD:-openmrs}
     volumes:
       - db-data:/var/lib/mysql
+    ports:
+      - 3306:3306
 
 volumes:
   openmrs-data: ~
-  db-data: ~
+  db-data: ~
```
Comment (jayasanka-sack) [inline docker/docker-compose-refqa-3x.yml:61]:
```suggestion
  db-data: ~

```

Comment (jayasanka-sack) [reply docker/docker-compose-refqa-3x.yml:61]: btw, are there any changes we need to make to this file? (I mean after the changes on the distro repo?)

Comment (pasindur99) [reply docker/docker-compose-refqa-3x.yml:61]: There were some changes in the [docker-compose-distro.yml](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/3.x/docker-compose-distro.yml) file with the latest commits. So I update the file to fit it with the latest commits in [3.x](https://github.com/openmrs/openmrs-distro-referenceapplication/tree/3.x) branch.

Comment (pasindur99) [reply docker/docker-compose-refqa-3x.yml]: Fixed EOL with [4dca1ad](https://github.com/openmrs/openmrs-test-3refapp/pull/3/commits/4dca1adc769581774ed8145ba09e1103ed87c4d7)

**Review by jayasanka-sack — APPROVED** [review]
Thanks @pasindur99 !

**Pull request merged**